it/v3/endpoints/SubmitSelfEmploymentBsasControllerISpec.scala

```diff
@@ -153,6 +196,10 @@ class SubmitSelfEmploymentBsasControllerISpec extends IntegrationBaseSpec {
         }
 
         val input = Seq(
+          (BAD_REQUEST, "INVALID_TAX_YEAR", BAD_REQUEST, TaxYearFormatError),
```

Comment:
Rather than adding in the TYS errors to `input` can we rename `input` -> `errors` and then also create a new `val extraTysErrors = Seq(...)` and update line 224 so `input` -> `(errors ++ extraTysErrors)`